features/fixtures/mazerunner/src/main/cpp/bugsnags.cpp

```diff
@@ -372,6 +372,24 @@ bool add_java_data(void *event_ptr) {
   return true;
 }
 
+bool add_smoke_data(void *event_ptr) {
+    bugsnag_event_add_metadata_string(event_ptr,
```

Comment:
As mentioned below, this call doesn't appear to be adding the corresponding metadata in the `CXXNotifySmokeScenario`.